app/assets/javascripts/discourse/app/instance-initializers/video-placeholder.js

```diff
@@ -0,0 +1,54 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import { iconHTML } from "discourse-common/lib/icon-library";
+
+export default {
+  initialize() {
+    withPluginApi("0.8.7", (api) => {
+      function _handleEvent(event) {
+        const svgIcon = event.target;
+        const overlay = svgIcon.parentElement;
+        const wrapper = overlay.parentElement;
+        const parentDiv = wrapper.parentElement;
```

Comment:
Would `event.target.closest(".element-class")` work? It would save a few lines (and a couple of variables). 

https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
Oh ya, I'll clean this up. It kind of just kept growing when I swapped out the svg asset.